### PR TITLE
VS 2015.2 Updates fixes (on proper branch)

### PR DESCRIPTION
**Pull request by ThePhD**
(Because github is a genius and you can't change PR branches with an edit)
Make code more compliant between compilers / with standard

### CODE REVIEW

File: lib/luwra/generic.hpp [hunk 1]
```diff
@@ -60,12 +60,8 @@ LUWRA_NS_END
...
 
-#ifdef _MSC_VER // Because VS C++
-	#define LUWRA_MEMBER(type, name) \
-		{#name, LUWRA_WRAP(##type::##name)}
-#else
-	#define LUWRA_MEMBER(type, name) \
-		{__STRING(name), LUWRA_WRAP(type::name)}
-#endif
+// This verions is standard and works with every compiler
+#define LUWRA_MEMBER(type, name) \
```
Comment (ThePhD) [inline lib/luwra/generic.hpp]: I sure hope so, at least

Comment (vapourismo) [inline lib/luwra/generic.hpp]: That doesn't work either.

Comment (ThePhD) [inline lib/luwra/generic.hpp]: Oh well, I tried: we'll keep the old version!

**Comment by vapourismo** [comment]
See if the current `vs2015` works for you.

**Comment by ThePhD** [comment]
It works. Thanks for all the hard work: sorry for VC++ being a butt! :+1:

**Comment by vapourismo** [comment]
Since the `vs2015` branch seems to work universally, I went ahead and merged it into master.
I intend to delete the `vs2015` branch, but will do so only, if you don't rely on it anymore (e.g. switched to `master`).

**Comment by ThePhD** [comment]
I transitioned. Throw the killswitch.

**Pull request merged**